website/content/en/preview/concepts/deprovisioning.md

```diff
@@ -49,16 +47,29 @@ There are both automated and manual ways of deprovisioning nodes provisioned by
     # Delete all nodes owned by a specific provisioner
     kubectl delete nodes -l karpenter.sh/provisioner-name=$PROVISIONER_NAME
     ```
+* **Provisioner Deletion**: Nodes are considered to be "owned" by the Provisioner that launched them. Karpenter will gracefully terminate nodes when a provisioner is deleted.
 
-Whether through node expiry or manual deletion, Karpenter seeks to follow graceful termination procedures as described in Kubernetes [Graceful node shutdown](https://kubernetes.io/docs/concepts/architecture/nodes/#graceful-node-shutdown) documentation.
-If the Karpenter controller is removed or fails, the finalizers on the nodes are orphaned and will require manual removal.
-
+### Automated Methods
+* **Emptiness**: Karpenter notes when the last workload (non-daemonset) pod stops running on a node. From that point, Karpenter waits the number of seconds set by `ttlSecondsAfterEmpty` in the provisioner, then Karpenter requests to delete the node. This feature can keep costs down by removing nodes that are no longer being used for workloads.
```

Comment:
We might be able to move things like use-cases into their own sections and just briefly mention the general idea of each of these methods here